functions/memorystore/redis/go.mod

```diff
@@ -0,0 +1,5 @@
+module visitcount
```

Comment:
Should probably have the full `github.com/GoogleCloudPlatform/golang-samples/...` path.

Reply:
Done.